app/components/primer/beta/banner.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= render(Primer::BaseComponent.new(**@wrapper_arguments)) do %>
+  <%= render(Primer::BaseComponent.new(**@system_arguments)) do %>
+    <% if @icon %>
+      <div class="Banner-visual">
+        <%= render(Primer::OcticonComponent.new(icon: @icon)) %>
+      </div>
+    <% end %>
+    <%= render(Primer::BaseComponent.new(**@message_arguments)) do %>
+      <p class="Banner-title" data-target="<%= catalyst_target(field: "titleText") %>"><%= content %></p>
+      <% if @description %>
+        <p><%= @description %></p>
+      <% end %>
+    <% end %>
+    <% if action %>
+      <div class="Banner-actions">
+        <%= action %>
+      </div>
+    <% end %>
+    <% if @dismissible %>
+      <div class="Banner-close">
+        <%= render(
+          Primer::Beta::IconButton.new(
+            scheme: :invisible,
+            icon: :x,
+            aria: { label: "Dismiss" },
+            data: { action: catalyst_action(event: "click", function: "dismiss") },
+            autofocus: true
```

Comment:
Should `Banner` component have `autofocus` too? Maybe this should be a conditional and set to `true` for `Flash` only?

Reply:
Good call! I actually moved all the `Banner` stuff into a [separate PR](https://github.com/primer/view_components/pull/1533) and have removed `autofocus` from it there.